bundle/compliance/lib/common/test.rego

```diff
@@ -45,6 +45,12 @@ test_array_contains_not_contains {
 	assert.is_false(array_contains(array, key))
 }
 
+test_array_contains_not_contains_substring {
+	array := ["a", "b", "ccc"]
+	key := "c"
+	assert.is_false(array_contains(array, key))
+}
+
```

Comment:
can we also add positive one?
```rego
test_array_contains {
	array := ["a", "b", "ccc"]
	key := "cccc"
	assert.is_true(array_contains(array, key))
}
```

Reply:
Happy to add another one but I don't think it covers a different case since we have this above:

```
test_array_contains {
	array := ["a", "b", "c"]
	key := "c"
	array_contains(array, key)
}
```